pkg/tsdb/cloudwatch/constants/metrics.go

```diff
@@ -395,7 +395,7 @@ var NamespaceMetricsMap = map[string][]string{
 	"AWS/EMRServerless":                  {"CancellingJobs", "CancelledJobs", "CPUAllocated", "FailedJobs", "IdleWorkerCount", "MaxCPUAllowed", "MaxMemoryAllowed", "MaxStorageAllowed", "MemoryAllocated", "PendingCreationWorkerCount", "PendingJobs", "RunningJobs", "RunningWorkerCount", "ScheduledJobs", "StorageAllocated", "SubmittedJobs", "SuccessJobs", "TotalWorkerCount"},
 	"AWS/ElasticTranscoder":              {"Billed Audio Output", "Billed HD Output", "Billed SD Output", "Errors", "Jobs Completed", "Jobs Errored", "Outputs per Job", "Standby Time", "Throttles"},
 	"AWS/EventBridge/Pipes":              {"Concurrency", "Duration", "EnrichmentStageDuration", "EnrichmentStageFailed", "EventCount", "EventSize", "ExecutionFailed", "ExecutionPartiallyFailed", "ExecutionThrottled", "ExecutionTimeout", "Invocations", "TargetStageDuration", "TargetStageFailed", "TargetStagePartiallyFailed", "TargetStageSkipped"},
-	"AWS/Events":                         {"DeadLetterInvocations", "Events", "FailedInvocations", "IngestionToInvocationStartLatency", "Invocations", "InvocationsFailedToBeSentToDlq", "InvocationsSentToDlq", "MatchedEvents", "ThrottledRules", "TriggeredRules"},
+	"AWS/Events":                         {"DeadLetterInvocations", "Events", "FailedInvocations", "Invocations", "InvocationAttempts", "InvocationsCreated", "InvocationsFailedToBeSentToDlq", "IngestiontoInvocationCompleteLatency", "IngestiontoInvocationStartLatency", "InvocationsSentToDlq", "MatchedEvents", "RetryInvocationAttempts", "SuccessfulInvocationAttempts", "ThrottledRules", "TriggeredRules", "PutEventsApproximateCallCount", "PutEventsApproximateFailedCount", "PutEventsApproximateSuccessCount", "PutEventsApproximateThrottledCount", "PutEventsEntriesCount", "PutEventsFailedEntriesCount", "PutEventsLatency", "PutEventsRequestSize", "PutPartnerEventsApproximateCallCount", "PutPartnerEventsApproximateFailedCount", "PutPartnerEventsApproximateThrottledCount", "PutPartnerEventsApproximateSuccessCount", "PutPartnerEventsEntriesCount", "PutPartnerEventsFailedEntriesCount", "PutPartnerEventsLatency"},
```

Comment:
Please sort all the metric names here for consistency. (I know not all the entries are sorted, but they're meant to be 😄 )

Reply:
Done!